IntrepidSwiftWisdom.podspec

```diff
@@ -1,22 +0,0 @@
-Pod::Spec.new do |s|
-  s.name          = "IntrepidSwiftWisdom"
```

Comment:
I think it is safe to remove this podspec because:
- the IntrepidSwiftWisdom spec still exists in the the spec repo
- the old tags still exist on https://github.com/IntrepidPursuits/swift-wisdom that reference IntrepidSwiftWisdom